model/project_ref.go

```diff
@@ -2244,55 +2260,52 @@ func (p *ProjectRef) GetGithubProjectConflicts() (GithubProjectConflicts, error)
 	return res, nil
 }
 
-// ValidateProjectCreation returns a boolean if you should surface the error or not.
-func ValidateProjectCreation(projectId string, config *evergreen.Settings, projectRef *ProjectRef) (bool, error) {
+// ValidateEnabledProjectsLimit takes in a merged project ref and validates if the current project is allowed to be enabled.
+// Returns a status code and error we are already at limit with enabled projects.
+func ValidateEnabledProjectsLimit(projectId string, config *evergreen.Settings, mergedRefToValidate ProjectRef) (int, error) {
 	if config.ProjectCreation.TotalProjectLimit == 0 || config.ProjectCreation.RepoProjectLimit == 0 {
```

Comment:
If mergedRefToValidate is disabled then we should return early.

Reply:
looks like we handle this by only calling the function if the mergedRefToValidate is enabled

Reply:
Maybe we can tweak the first comment to something like "and validates project limits, assuming the given project is going to be enabled". Also isn't quite accurate since it says it takes in 1 project ref.

Reply:
done